docs/Actions.md

```diff
@@ -290,19 +291,25 @@ import { useGetOne, useRecordContext } from 'react-admin';
 
 const UserProfile = () => {
     const record = useRecordContext();
-    const { data, isLoading, error } = useGetOne(
+    const { data, isPending, error } = useGetOne(
         'users',
         { id: record.id },
         { onSettled: (data, error) => console.log(data, error) }
     );
-    if (isLoading) { return <Loading />; }
+    if (isPending) { return <Loading />; }
     if (error) { return <p>ERROR</p>; }
     return <div>User {data.username}</div>;
 };
 ```
 
 We won't re-explain all these options here, but we'll focus on the most useful ones in react-admin. 
 
+- `onSuccess(data, variables, context)`: The `onSuccess` function is called when the query returns. It receives the query data, the [query variables](https://tanstack.com/query/latest/docs/react/guides/query-functions#query-function-variables) and the [query context](https://tanstack.com/query/latest/docs/react/guides/query-functions#queryfunctioncontext).
+
+- `onError(error, variables, context)`: The `onSuccess` function is called when the query fails. It receives the error, the [query variables](https://tanstack.com/query/latest/docs/react/guides/query-functions#query-function-variables) and the [query context](https://tanstack.com/query/latest/docs/react/guides/query-functions#queryfunctioncontext).
+
+- `onSettled(data, error, variables, context)`: The `onSuccess` function is called after the query either succeeded or failed. It receives the query data (can be `undefined` if the query failed), the error (can be `undefined` when the query succeeded), the [query variables](https://tanstack.com/query/latest/docs/react/guides/query-functions#query-function-variables) and the [query context](https://tanstack.com/query/latest/docs/react/guides/query-functions#queryfunctioncontext).
```

Comment:
I think we should add an example usage for each of these, with real use cases (e.g. notification, cache invalidation, etc)